src/Widgets/FormStepContainerWidget/components/ConditionalDropdownHeader.tsx

```diff
@@ -24,13 +24,15 @@ export const DropdownHeader = Scrivito.connect(
           onChange={onChangeSelected}
         >
           <EmptyOption />
-          {widget.get("conditions").map((condition, index) => (
-            <DropdownOption
-              value={condition.get("title")}
-              id={condition.id()}
-              key={index}
-            />
-          ))}
+          {widget
+            .get("conditions")
+            .map((condition: Scrivito.Widget, index: number) => (
+              <DropdownOption
+                value={condition.get("title") as string}
+                id={condition.id()}
+                key={index}
```

Comment:
`key={index}` could be `key={condition.id()}`

Reply:
Done